### DISCUSSION TITLE
Custom Dataset with AI-VAD

### DISCUSSION THREAD

**Discussion by arjun-kg** [open-edge-platform/anomalib#1707] [Q&A]
Hi! I am interested in using AI-VAD with a different dataset, but it seems only Shanghai and UCSDPed Datamodules are present currently (unlike the image case where `Folder` can be used for custom datasets). Right now, is there a way to customize UCSDPed datamodule for eg. and use it with a custom dataset? Or are some things hardcoded for those particular datasets? 

I'm also not sure if the data format can match, I have normal and abnormal clips (no masks, just video labels), I want to do classification.

**Comment by samet-akcay** [comment]
@arjun-kg, as you mentioned custom video data support is not yet available. One alternative would be to convert your dataset format to `UCSDPed` or other supported formats. 

Custom video data is on our roadmap but didn't have time implement yet.
